src/main/core/Controller/APINew/User/UserController.php

```diff
@@ -307,6 +312,7 @@ public function resetPasswordAction(Request $request): JsonResponse
         foreach ($users as $user) {
             if ($this->checkPermission('ADMINISTRATE', $user)) {
                 $this->mailManager->sendForgotPassword($user);
+                $this->eventDispatcher->dispatch(SecurityEvents::FORGOT_PASSWORD, ForgotPasswordEvent::class, [$user]);
```

Comment:
Should be dispatched from the `sendForgotPassword()` method. Not in the controller.

Reply:
Done, thx :)